src/tallies/tally.cpp

```diff
@@ -906,7 +906,7 @@ void reduce_tally_results()
     auto& tally {model::tallies[i_tally]};
 
     // Get view of accumulated tally values
-    auto values_view = xt::view(tally->results_, xt::all(), xt::all(), RESULT_VALUE);
+    auto values_view = xt::view(tally->results_, xt::all(), xt::all(), (int)RESULT_VALUE);
```

Comment:
[Use a named cast](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-casts-named)

Reply:
Still weaning off my old C ways, thanks for this tip!